corehq/apps/reports/filters/users.py

```diff
@@ -473,6 +475,28 @@ def get_default_selections(self):
         return [('t__6', _("[Web Users]"))]
 
 
+class PropertyFilter(BaseSingleOptionFilter):
+    label = ugettext_noop('Modified Property')
+    default_text = ugettext_noop('Select Property')
+    slug = 'property'
+
+    @property
+    def options(self):
+        if domain_has_privilege(self.domain, privileges.APP_USER_PROFILES):
+            user_data_label = _("Profile or User Data")
+        else:
+            user_data_label = _("User Data")
+        return [
+            ("is_active", _('Active / Deactivated')),
+            ("email", _('Email')),
+            ("language", _('Language')),
+            ("location_id", _('Location')),
```

Comment:
this is primary location for just the mobile worker yes?

`location_id` and `assigned_location_ids` is only set for CommCareUser and not for WebUser since for web user these are stored on Domain Membership.
we should keep that clear here.
"Mobile User Primary Location" or "Mobile User Location" (since mobile user only has one location so 'Primary' isn't really needed)

Reply:
Updated in b1ab22c5104 to specify this is just the primary location and only for mobile users.

Is there a plan for adding domain membership changes? I can't remember if we discussed this already, but looking at it now, it seems odd that web user location changes show up in the message but don't have anything in the changes dict.

![Screen Shot 2021-07-16 at 8 32 58 PM](https://user-images.githubusercontent.com/1486591/126020124-5a55c7a4-59ab-4722-99f2-14c07a1c1de0.png)

Reply:
> Is there a plan for adding domain membership changes?

No. No plans to include changes in related models as direct property changes.

> I can't remember if we discussed this already, but looking at it now, it seems odd that web user location changes show up in the message but don't have anything in the changes dict.

Currently the report is showing exactly what is in the model, and the model keeps "related model's change messages" separately. The UI definitely can show it however it's better for UI/UX.

Reply:
Hmm, I think it'll be confusing for users that they can only see locations for mobile users, not web users. But we don't need to deal with that right now.

Reply:
searching would be limited to mobile workers location but users should be able to see locations on both mobile and web in the report messages.